src/backend/d3d12/TextureD3D12.cpp

```diff
@@ -0,0 +1,133 @@
+// Copyright 2017 The NXT Authors
```

Comment:
Yea, I had a reduced version here because the binding model PR wasn't merged yet. Updated.